container/features/src/main/resources/features-minion.xml

```diff
@@ -47,5 +47,15 @@
 	  <bundle>blueprint:mvn:org.opennms.features.minion/core-impl/${project.version}/xml/blueprint-heartbeat</bundle>
 	  <feature>minion-core-api</feature>
 	</feature>
+
+    <feature name="opennms-detector-minion" description="OpenNMS :: Minion :: Detector" version="${project.version}">
```

Comment:
This feature should be renamed to *opennms-detector-simple* and moved into the other features.xml file since it is not Minion specific.